flaml/automl/task/task.py

```diff
@@ -1,17 +1,11 @@
 from abc import ABC, abstractmethod
 from typing import TYPE_CHECKING, List, Optional, Tuple, Union
-
 import numpy as np
-import pandas as pd
+from flaml.automl.data import DataFrame, Series
```

Comment:
Maybe `from flaml.automl.data import psDataFrame, psSeries, DataFrame, Series` and add `psDataFrame, psSeries` to below code.